app/components/pages/Contact.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
## Code Quality - Nitpicks

React Components should be in files with the extension .jsx